libres/lib/python/enkf_defaults.cpp

```diff
@@ -0,0 +1,11 @@
+#include <ert/enkf/enkf_defaults.hpp>
+#include <ert/python.hpp>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+namespace py = pybind11;
+
+void enkf_defaults(py::module_ m) {
+    m.def("default_gen_kw_export_name",
```

Comment:
You can use `m.add_object` to add a constant. Eg:
```c++
void enkf_defaults(py::module_ m) {
    m.add_object("DEFAULT_GEN_KW_EXPORT_NAME", DEFAULT_GEN_KW_EXPORT_NAME);
}
```

Is equivalent with:
```py
# enkf_defaults.py
DEFAULT_GEN_KW_EXPORT_NAME = <Value of the macro>
```

Reply:
```
void enkf_defaults(py::module_ m) {
    m.add_object("DEFAULT_GEN_KW_EXPORT_NAME", DEFAULT_GEN_KW_EXPORT_NAME);
}
```

Cant set this up with a macro, suggested way is to make a function it seems?

Reply:
```
error: no viable conversion from 'const char [11]' to 'pybind11::handle'
        m.add_object("DEFAULT_GEN_KW_EXPORT_NAME", DEFAULT_GEN_KW_EXPORT_NAME);
```

Reply:
Exposing it as a function works though.

Reply:
I forgot to add that you might need to wrap the value: `py::str(DEFAULT_GEN_KW_EXPORT_NAME)` sorry.